session/handler_test.go

```diff
@@ -601,9 +601,10 @@ func TestHandlerSelfServiceSessionManagement(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, http.StatusNoContent, res.StatusCode)
 
-		actualOthers, err := reg.SessionPersister().ListSessionsByIdentity(ctx, i.ID, nil, 1, 10, uuid.Nil)
+		actualOthers, total, err := reg.SessionPersister().ListSessionsByIdentity(ctx, i.ID, nil, 1, 10, uuid.Nil)
 		require.NoError(t, err)
 		require.Len(t, actualOthers, 3)
+		require.Equal(t, int64(3), total)
```

Comment:
I'm breaking the testing pattern established by the other assertions in these tests by setting the expected value as the second param. The [docs for the `require.Equal`](https://pkg.go.dev/github.com/stretchr/testify/require#Equal) specify that the second param be the expected value, so I just followed those. If you'd prefer that I follow the repo convention rather than the `require` convention, I'm happy to change this around.

Reply:
There is no "repo convention", it should be the expected value first. There were just mistakes in the past :wink: